templates/addons/calico-ipv6.yaml

```diff
@@ -314,6 +314,268 @@ status:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
```

Comment:
@jsturtevant these are all the changes made in the ipv6 manifest by the 3.20 -> 3.22 update. Let's see if this PR repros the issue that was happening in #2021

Reply:
Interesting... I don't see anything that jumps out as different configuration